Runtime/Data/Collection/ObservableList.cs

```diff
@@ -16,9 +18,27 @@ public abstract class ObservableList<TElement, TEvent> : MonoBehaviour where TEv
         /// <summary>
         /// The collection to observe changes of.
         /// </summary>
-        [SerializeField, DocumentedByXml]
-        protected List<TElement> elements = new List<TElement>();
+        [Serialized]
+        [field: DocumentedByXml]
+        protected List<TElement> Elements { get; set; } = new List<TElement>();
 
+        /// <summary>
+        /// The pointer for the current index being specified in the collection.
```

Comment:
```suggestion
        /// The index to use in methods specifically specifying to use it. In case this index is out of bounds for the collection it will wrap around.
```